tests/specs/mcd-structured/verification.k

```diff
@@ -265,7 +265,7 @@ module LEMMAS-MCD [symbolic]
   // Arithmetic
   // ########################
 
-    rule X <=Int maxUInt48 &Int (X +Int Y) => false
+    rule X <=Int #asWord ( #range ( #buf ( 32, X +Int Y ), 26, 6 ) ) => false
```

Comment:
The `maxUInt48 &Int (X +Int Y)` term gets transformed into the other one, so this very specific simplification needs to follow.